src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/actions/WorkOnBacklog.kt

```diff
@@ -31,7 +32,7 @@ class WorkOnBacklog private constructor(
     }
 
     private val logger: Logger = LogManager.getLogger(this::class.java)
-    private val newSprintName = "New Sprint Name"
+    private val newSprintName = RandomStringUtils.randomAlphabetic(10)
```

Comment:
If this is random part of the test we should reuse `seededRandom`

Reply:
I started with UUID, but it's too long (sprint name can be 30 characters long at most), then I switched to this solution.

Reply:
I'm not sure if I was understood correctly.

In the past a design decision was made in JPT to make VU execution repeatable. The idea to do that was by not using unseeded randomisation.

In the constructor of `WorkOnBacklog` you can find a `SeededRandom` field that was supposed to be used for this purpose. It contains `java.util.Random` instance, which can be used for randomisation of numbers. Each VU thread has it's `SeededRandom` with different seed, which is generated from the original seed, so random sequence should be unique between VUs and are repeatable.

I don't see that we have out of the box mechanism to generate text out of `Random` or `SeededRandom`, however creating it should be pretty easy and could be promoted in the future to `jira-actions` module.

Reply:
We don't need text per se, we can generate random using `SeededRandom` number and glue it with sprint name

Reply:
Yo, this is randomizations is done once per `WorkOnBacklog` instance. This should be done every time the interaction is called.